Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxy.cpp

```diff
@@ -260,13 +260,15 @@ RefPtr<Image> RemoteGraphicsContextGLProxy::videoFrameToImage(WebCore::VideoFram
     if (isContextLost())
         return { };
 
-    RefPtr<NativeImage> nativeImage;
 #if PLATFORM(COCOA)
+    RefPtr<NativeImage> nativeImage;
     callOnMainRunLoopAndWait([&] {
         nativeImage = protectedVideoFrameObjectHeapProxy()->getNativeImage(frame);
     });
-#endif
     return BitmapImage::create(WTFMove(nativeImage));
+#endif
+
+    return GraphicsContextGL::videoFrameToImage(frame);
```

Comment:
My thought (maybe I was not clear earlier) was to put the alternative inside the `#else` part of the guard:

```suggestion
#else
    return GraphicsContextGL::videoFrameToImage(frame);
#endif
```

The compiler should optimize away the second `return` anyway, so no big deal.